serf/internal_query.go

```diff
@@ -29,6 +29,13 @@ const (
 
 	// listKeysQuery is used to list all known keys in the cluster
 	listKeysQuery = "list-keys"
+
+	// maxListKeyFactor is used to compute the max number of keys in a list key
+	// response. eg 1024/25 = 40. a message with max size of 1024 bytes cannot
+	// contain more than 40 keys. There is a test
+	// (TestSerfQueries_estimateMaxKeysInListKeyResponse) which does the
+	// computation and in case of changes, the value can be adjusted.
+	maxListKeyFactor = 25
```

Comment:
nit: is it worth mentioning that 25 is derived from the length of an AES 16 byte key in base64? I might even call it `minEncodedKeyLength` or something but up to you. Keys are not always 16 bytes necessarily - memberlist supports 32 byte keys to _in theory_ in practice we probably only use 16 bytes but that is the minimum which is what we care about for our estimate.